app/models/manageiq/providers/embedded_terraform/automation_manager/amazon_credential.rb

```diff
@@ -0,0 +1,68 @@
+class ManageIQ::Providers::EmbeddedTerraform::AutomationManager::AmazonCredential < ManageIQ::Providers::EmbeddedTerraform::AutomationManager::CloudCredential
+  COMMON_ATTRIBUTES = [
+    {
+      :component  => 'text-field',
+      :label      => N_('Access Key'),
+      :helperText => N_('AWS Access Key for this credential'),
+      :name       => 'userid',
+      :id         => 'userid',
+      :isRequired => true,
+      :validate   => [{:type => 'required'}],
+    },
+    {
+      :component  => 'password-field',
+      :label      => N_('Secret Key'),
+      :helperText => N_('AWS Secret Key for this credential'),
+      :name       => 'password',
+      :id         => 'password',
+      :type       => 'password',
+      :isRequired => true,
+      :validate   => [{:type => 'required'}],
+    },
+  ].freeze
+
+  EXTRA_ATTRIBUTES = [
+    {
+      :component  => 'password-field',
+      :label      => N_('STS Token'),
+      :helperText => N_('Security Token Service(STS) Token for this credential'),
+      :name       => 'security_token',
+      :id         => 'security_token',
+      :type       => 'password',
+      :maxLength  => 1024
+    },
+    {
+      :component  => 'text-field',
+      :label      => N_('Region'),
+      :helperText => N_('AWS Region where the provider will operate. The Region must be set.'),
```

Comment:
Do we need to say "The region must be set."? I don't see this for other `isRequired => true` fields.  We also highlight the field as required if you don't provide a value.

Reply:
@putmanoj I see you resolved this comment but didn't change the code.  I agree with @jrafanie saying `The Region must be set` is redundant when we already have the field required.

Reply:
To be resolved in in https://github.com/ManageIQ/manageiq-providers-embedded_terraform/pull/21